td.vue/package.json

```diff
@@ -73,9 +77,11 @@
     "portal-vue": "^2.1.7",
     "sass": "^1.32.12",
     "sass-loader": "^10.1.1",
+    "spectron": "15.0.0",
```

Comment:
latest version of spectron is 19.0.0, can we use this latest version?

Reply:
Done! Thank you